### DISCUSSION TITLE
suddenly my server function is 138 MB, without any changes

### DISCUSSION THREAD

**Discussion by PeterF2170** [vercel/community#1052] [Help] [answered]
I am developing an SSR Nuxt app. I'm using vercel-builder to build my serverside part.

Until today it worked everytime, but now the limit of the server function is exceeded. How can i decrease the function size?

The size of the main-branch index function is 46MB.
If I create a new branch and try to deploy the branch on vercel, the size of the index function suddenly is 138 MB.
How is this even possible? What influences the function size and how can it be decreased?

My vercel.json:

```{
    "version": 2,
    "builds": [
      {
        "src": "nuxt.config.js",
        "use": "@nuxtjs/vercel-builder",
        "config": {
          "serverFiles": ["package.json"],
          "generateStaticRoutes": true
        }
      }
    ]
  }
```

Screenshots of the Main-Branch Function:

![image](https://user-images.githubusercontent.com/56222670/201871260-8a96f157-5623-4fdc-949c-5ad3738dc57e.png)

I only deleted the End of line, so no line of code was changed:

![image](https://user-images.githubusercontent.com/56222670/201874948-62d1a280-f36b-460e-8bad-4ecdaa31d1d5.png)

And suddenly the size soars to 138 MB:

![image](https://user-images.githubusercontent.com/56222670/201874824-a309cf12-5c34-4730-b84e-0e23be84f8e2.png)

**Comment by swarnava** [comment, marked as answer]
It is likely related to [@nuxt/vercel-builder](https://github.com/nuxt/vercel-builder/issues/727#issuecomment-1311359085) regression that has introduced in recent release, can you try older version?

**Reply by PeterF2170** [reply]
Thanks, now I'm using `vercel-builder@0.22.1` in my vercel.json, which reduces the size to 50.32 MB. Any suggestions how to save even more size?

**Reply by swarnava** [reply]
It's hard to tell without looking at the code. You can run `vercel build` locally with the old version of the builder and the new version of the builder (you can select the version in `vercel.json`). Then compare the output of `.vercel/output/functions` directory
